README.md

```diff
@@ -0,0 +1,34 @@
+# BETTER BUILDS
+This React website is for users to submit and review PC builds that will be posted to PostgreSQL database. Users can also edit their reviews and vote on the reviews of other users with Express backend, running on a NodeJS environment.
+## Contributors: 
+Nick DeRis - github.com/dericholas
+Nick Kwiatkowski - github.com/nickkwiat
+Robert Le - github.com/fuzzyphantasm
+Corey Pierce - github.com/cornfrog
+Marlon Sanchez - github.com/MarlonS419 
+## Installation
+```
+git clone https://github.com/MarlonS419/better-builds
+```
+then Navigate to <localhost:3000/> in your browser
+## Development
+```
+# opens directory
+cd better-builds
+# installs dependencies
+yarn install
+# to create the database
+createdb better-builds_development 
+# to create tables
+yarn migrate:latest
+# starts the program
+yarn start
+```
+## TODO
```

Comment:
you don't need to fill this section out right now as you are just starting on the app, you can plan to come back to this to do section with your group next week once more functionality has been built out